core/assets/bundles/bundle_fr.properties

```
@@ -599,14 +622,17 @@ setting.difficulty.insane = Extrême
 setting.difficulty.name = Difficulté:
 setting.screenshake.name = Tremblement de l'écran
 setting.effects.name = Afficher les effets
-setting.destroyedblocks.name = Display Destroyed Blocks
-setting.conveyorpathfinding.name = Conveyor Placement Pathfinding
+setting.destroyedblocks.name = Afficher les Blocs Détruits
+setting.conveyorpathfinding.name = Recherche de Chemin pour le Placement de Convoyeurs
 setting.sensitivity.name = Sensibilité de la manette
 setting.saveinterval.name = Intervalle des sauvegardes auto
 setting.seconds = {0} secondes
+setting.blockselecttimeout.name = Délai d'attente de sélection de bloc
```

First letters of each word should be in uppercase.
```suggestion
setting.blockselecttimeout.name = Délai d'Attente de Sélection de Bloc
```
  
*See also: https://github.com/Anuken/Mindustry/blob/7f2a529c081cabd191e53e9358ecf029648a3194/core/assets/bundles/bundle.properties#L630*